onnxruntime/core/session/inference_session.cc

```diff
@@ -87,6 +160,38 @@ inline std::basic_string<T> GetCurrentTimeString() {
   return std::basic_string<T>(time_str);
 }
 }  // namespace
+struct CustomOpKernel : OpKernel {
+  CustomOpKernel(const OpKernelInfo& info, OrtCustomOp& op) : OpKernel(info), op_(op) {
```

Comment:
Should this have ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE?

Reply:
Yes